presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/BenchmarkReorderJoins.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
not for the case in the benchmark because that was a fully connected graph.  when you start join enumeration you don't know how connected your graph is because of equality inference, so it seemed like the safest measurement to take.